test/runner/tob.ts

```diff
@@ -137,7 +135,7 @@ function diagnostics2md(diagnostics: any) {
   let diagnosticString = '';
   if (diagnostics.length) {
     diagnostics.forEach(function (a: any) {
-      diagnosticString += '✖ ' + a.message + ' - start: ' + a.start + ', end: ' + a.length;
+      diagnosticString += '✖ ' + a[1] + ' - start: ' + a[2] + ', end: ' + a[3];
```

Comment:
* a[0] => code
* a[1] => message
* a[2] => start
* a[3] => end

was it right?

Reply:
yes. The code is either lexer or parser. Not important here. 2nd arg is message. then start and end

Reply:
.length is wrong. should be 'a .end'. same as you see on the CST node